### PR TITLE
bugfix: modify tty_win info in runtime when handling ResizePtyRequest

### PR DESCRIPTION
**Pull request by YushuoEdge**
Currently, we only create the new exec process in runtime, this will cause error when the following requests needing to be handled:

- Task: exec process
- Task: resize process pty
- ...

The agent do not do_exec_process when we handle ExecProcess, thus we can not find any process information in the guest when we handle ResizeProcessPty. This will report an error.
    
In this commit, the handling process is modified to the:
    * Modify process tty_win information in runtime
    * If the exec process is not running, we just return. And the truly pty_resize will happen when start_process

Fixes: #6248

### CODE REVIEW

**Comment by YushuoEdge** [comment]
@quanweiZhou @Tim-0731-Hzt  PTAL!

File: src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container.rs [hunk 1]
```diff
@@ -344,20 +344,35 @@ impl Container {
...
+                return Ok(());
+            }
+        } else {
+            warn!(
+                sl!(),
+                "could not find process {} in container {}",
+                process.exec_id(),
+                process.container_id()
+            );
+            return Ok(());
```
Comment (Tim-0731-Hzt) [inline src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container.rs:372]: why should we return OK if the process is invalid

**Review by quanweiZhou — APPROVED** [review]
Thanks, @YushuoEdge LGTM.

File: src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container.rs [hunk 2]
```diff
@@ -344,20 +344,35 @@ impl Container {
...
-            .tty_win_resize(agent::TtyWinResizeRequest {
-                process_id: process.clone().into(),
-                row: height,
-                column: width,
-            })
-            .await
-            .context("resize pty")?;
-        Ok(())
+
+        if process.exec_id.is_empty() {
```
[the remark this answers is not in the document]
Comment (YushuoEdge) [reply src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container.rs:353]: Actually, I wrote it here just imitating what is wrote in agent, see: https://github.com/kata-containers/kata-containers/blob/main/src/agent/src/sandbox.rs#L239-L252

The request comes from upper layer, and maybe there are some cases the request did not contain eid.

File: src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container.rs (hunk 1 above)
Comment (YushuoEdge) [reply src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container.rs:372]: I don't think this is on the critical path, so I just return `Ok(())` here, like in the above, we will not return `Err(...)` when the container is not in Running state.

Comment (YushuoEdge) [reply src/runtime-rs/crates/runtimes/virt_container/src/container_manager/container.rs]: OK, fixed.

**Review by Tim-0731-Hzt — APPROVED** [review]

**Pull request merged**